roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -32,6 +32,20 @@
 
 - include_tasks: determine_version.yaml
 
+- set_fact:
+    full_restart_cluster: True
+  when:
+    - _es_installed_version is defined
+    - __es_version.split('.')[0] | int >= 5
```

Comment:
Should this be hardcoded to 5?  What if we need to go from 5->6 ?  Or do you think its better to wait until that time comes?

Reply:
yeah, I wasn't entirely sure... I think for now its safe to do that since its only in this one location. We may be able to get away with not even having this line, though we would probably want to safeguard that we aren't coming from a `1.x` version then (that was the intent with this)

Reply:
I removed this line, when the time comes and we want to safeguard against going from 1.x -> 5.x we should have an `assert` earlier in the installation process to prevent the installation from completing not try to stop the restart. We want to restart any time the major version increments.